### PR TITLE
external/webclient : add chunked encoding and refactoring callback flow

### PR DESCRIPTION
**Pull request by JeonginKim**
Add chunked type encoding processing
Callback is called as soon as data is received

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

File: external/webclient/webclient.c [hunk 1]
```diff
@@ -1123,7 +1124,6 @@ static pthread_addr_t wget_base(void *arg)
 	int sockfd = -1;
 	int ret;
 	int buf_len, sndlen, len;
```
Comment (sangwon03) [inline external/webclient/webclient.c]: do we need 'buf_len' variable?

**Comment by seinfra** [comment]
Code Rule Check OK.

File: external/webclient/webclient.c (hunk 1 above)
Comment (JeonginKim) [reply external/webclient/webclient.c]: It is used when sending, but not when receiving. It's corrected.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**